README.md

```diff
@@ -68,12 +68,20 @@ The easiest way to install Spyder is:
 
 ### On Windows:
 
-- Or using one of these scientific Python distributions:
+- You can use `pip`:
+
+    pip install spyder
```

Comment:
Please add this entry after the Scientific distributions one. It's far more easier to install those distros, so I want our users to see it as the first entry :-)